CSV_filter.java

```diff
@@ -0,0 +1,44 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CSV_filter {
```

Comment:
Also here, better use class names written in CamelCase instead of snake_case.
 ```suggestion
public class CsvFilter {
```

Reply:
Alles Klar！